src/pages/[userId]/prospects.js

```diff
@@ -1,7 +1,51 @@
 import { useRouter } from 'next/router';
 
+import Grid from '../../utils/Grid';
+
+import Prospect from '../../components/Prospect';
+
 export default function Prospects() {
   const router = useRouter();
   const { userId } = router.query;
-  return <p>Prospects for {userId}</p>;
+
+  const prosArr = [
+    {
+      id: 1,
+      first_name: 'Jon',
+      last_name: 'Smith',
+      notes: 'contacted on 5/17',
+    },
+    {
+      id: 2,
+      first_name: 'Matt',
+      last_name: 'Clark',
+      notes: 'need to contact',
+    },
+    {
+      id: 3,
+      first_name: 'Mitchell',
+      last_name: 'Pavel',
+      notes: 'No contact list',
+    },
+    {
+      id: 4,
+      first_name: 'Bob',
+      last_name: 'Chin',
+      notes: 'contacted on 05/10',
+    },
+  ];
+
+  const prospects = prosArr.map((prospect) => <Prospect key={prospect.id} donor={prospect} />);
```

Comment:
Change prop name `donor` to `prospects`.

Also lets avoid abbreviating variable names, like `prosArr` because it can easily get confusing later on. One way to get around this naming issue would be to:
- name the array of prospects data `prospects` instead of `prosArr` (we'll do the same thing when we're actually retrieving this from the db)
- like my comment on the pledges page, we can move the array of `Prospect` components created on line 38 straight into the markup on line 48. So instead of `{pledges}` we have `prospects.map((prospect) => <Prospect key={prospect.id} donor={prospect} />)`. Then we avoid needing to create another variable name.